### ISSUE TITLE
StatusControl.xaml.cs and doing states in code

### ISSUE THREAD

**Issue by crutkas** [Microsoft/WindowsTemplateStudio#610]
Looking at https://github.com/Microsoft/WindowsTemplateStudio/pull/604#pullrequestreview-48072723 and noticed we're setting styles in code, with accessibility and tracking stuff down, i feel like these should be done via a style in code.

Have the code behind set the state, not the actual styles on individual elements.

**Comment by mvegaca** [comment]
Added on #676
